tests/test_wps_generate_climos.py

```diff
@@ -87,7 +86,7 @@ def build_params(netcdf, kwargs):
                 "convert_longitudes": "False",
                 "split_vars": "False",
                 "split_intervals": "False",
-                "dry_run": "True",
+                "dry_run": "False",
```

Comment:
Running `ce-dp`'s [input_check](https://github.com/pacificclimate/climate-explorer-data-prep/blob/f888df34322fa4c7ba511970fb91c5e251dbab52/dp/generate_climos.py#L42) fails when used on the one-week's worth of `pr`, `tasmin`, and `tasmax` data because they have no climo periods, though they work fine when running the full process. If you want to make sure dry runs work, I can instead restrict the opendap testdata to not have these sets.

Reply:
You can leave this as is, thanks for the explanation!

Reply:
I looked at `ce-dp`'s `generate_climos` again, and I actually think I should remove those test cases. Since those files have no climo periods, [this for loop](https://github.com/pacificclimate/climate-explorer-data-prep/blob/f888df34322fa4c7ba511970fb91c5e251dbab52/dp/generate_climos.py#L110) doesn't get run, so no output is being generated.

Reply:
Ok, good find.